haskell-src/Concordium/Logger.hs

```diff
@@ -1,10 +1,12 @@
 {-# LANGUAGE DerivingVia #-}
 {-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE StandaloneDeriving #-}
```

Comment:
I'm not sure where this was needed in this file.

Reply:
Removed.